hr_employee_firstname/__manifest__.py

```diff
@@ -8,7 +8,7 @@
     "Fekete Mihai (Forest and Biomass Services Romania), "
     "Onestein, "
     "Odoo Community Association (OCA)",
-    "maintainer": "Savoir-faire Linux",
+    "maintainers": ["Savoir-faire Linux", "luisg123v"],
```

Comment:
```suggestion
    "maintainers": ["Savoir-faire Linux"],
```

please add your contributor in file `readme/CONTRIBUTORS.rst`

Reply:
@Saran440 why is that?

Maintainer and contributors are not the same.

Reply:
The change is so small that it does not need to be added to CONTRIBUTORS

But it is good to be maintainers in order to be notified of changes

Reply:
oh, sorry i misunderstand that it author field.